ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/ThreadNet/Network.hs

```diff
@@ -307,12 +331,18 @@ runThreadNetwork systemTime ThreadNetworkArgs
     -- fork the directed edges, which also allocates their status variables
     let uedges = edgesNodeTopology nodeTopology
     edgeStatusVars <- fmap (Map.fromList . concat) $ do
+      -- assume they all use the same CodecConfig
```

Comment:
Just a heads-up, as of #2336 the `CodecConfig` is just another field in the top-level config (and we'll want to vary it from node to node once we start decontamination).

Reply:
Ah, merged just in time 😛